plugins/make_deb

```diff
@@ -236,6 +236,13 @@ EOF
             mkdir -p usr/lib
             cp -R --preserve=links /usr/lib/libbiodaq* usr/lib
         fi
+        if [ "${plugin_name}" = "gcp-gateway" -o "${plugin_name}" = "gcp" ]; then
+            cd -
+            mkdir -p usr/local/lib
+            cp -R --preserve=links /usr/local/lib/libjwt* usr/local/lib
+            cp -R --preserve=links /usr/local/lib/libjansson* usr/local/lib
+            cp -R --preserve=links /usr/local/lib/libpaho* usr/local/lib
```

Comment:
Looks only `libmqttv3cs.so - "classic" / synchronous with SSL` is required. May be we can copy 
`libmqttv3*s.so` to copy asynchronous and classic" / synchronous both will SSL. **No harm copying all**